### PR TITLE
Feature/email notifications

### PR DESCRIPTION
**Pull request by bigardone**
Now emails are delivered when:
- User creates a new vacation request.
- Admin approves a vacation request.
- Admin rejects a vacation request.
- Admin approves/rejects single requested days.

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "info@holidays.diacode.com"
```
Comment (hopsor) [inline app/mailers/application_mailer.rb]: can we change it to `holidays@diacode.com` ?

Comment (bigardone) [inline app/mailers/application_mailer.rb]: Of course!

File: app/mailers/vacation_requests_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class VacationRequestsMailer < ApplicationMailer
+  def admin_notification(vacation_request)
+    @vacation_request = vacation_request
+    @user = vacation_request.user
+
+    mail to: User.admin_emails, subject: "New vacation request from #{@user.full_name}"
+  end
+
+  def approved_days(user, days)
+    @days = Array days
```
Comment (hopsor) [inline app/mailers/vacation_requests_mailer.rb:10]: first time I see this. Wouldn't be `@days = Array.new days` ?

Comment (bigardone) [inline app/mailers/vacation_requests_mailer.rb:10]: http://ruby-doc.org/core-2.2.0/Array.html#class-Array-label-Creating+Arrays

> An array can also be created by using the Array() method, provided by Kernel, which tries to call to_ary, then to_a on its argument.

**Pull request merged**